src/Helper/Fields/Field_Product.php

```diff
@@ -62,7 +62,7 @@ public function html( $value = '', $label = true ) {
 			if ( in_array( $this->field->get_input_type(), [ 'radio', 'select' ], true ) ) {
 				$html .= Kses::parse( $value['name'] . ' - ' . $value['price'] );
 			} else {
-				$html .= esc_html( $value['price'] );
+				$html .= esc_html( $value['price'] ) . ' (' . $value['quantity'] . ' ' . __( 'Qty', 'gravity-forms-pdf-extended' ) . ') ';
```

Comment:
@jestonihpi couple things here:

1. The quantity should not be included in the output if there is a quantity field in the form that links to this product. Otherwise the PDF will display the quantity twice (one in the Product field and once in the Quantity field). 

2. The quantity should not be included in the output if the quantity has been disabled in the Product field

3. The new string you added needs to go _inside_ the `esc_html()` function.

Reply:
Copy  @jakejackson1 , thanks!